{"uid":"7cf890e9dd9ecea1","name":"Checks both cards button is not disabled","fullName":"Checks both cards button is not disabled","historyId":"ddf65edc1027d3447a3ce8590354e5d2","time":{"start":1681596526831,"stop":1681596528081,"duration":1250},"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"TypeError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (http://localhost:4173/src/bootstrap.tsx:5:10)","flaky":true,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"TypeError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (http://localhost:4173/src/bootstrap.tsx:5:10)","steps":[{"name":"\"before each\" hook","time":{"start":1681596526831,"stop":1681596527028,"duration":197},"status":"passed","steps":[{"name":"visit http://localhost:4173","time":{"start":1681596526906,"stop":1681596527028,"duration":122},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"http://localhost:4173 request","time":{"start":1681596527028,"stop":1681596527028,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"http://localhost:4173 request","time":{"start":1681596527028,"stop":1681596527028,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":3,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},{"name":"waitUntil {}","time":{"start":1681596527030,"stop":1681596528080,"duration":1050},"status":"passed","steps":[{"name":"uncaught exception TypeError: ReactDOM.createRoot is not a function","time":{"start":1681596527032,"stop":1681596528080,"duration":1048},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"get button","time":{"start":1681596528079,"stop":1681596528079,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"assert expected **undefined** to have a length of **2** but got **0**","time":{"start":1681596528079,"stop":1681596528079,"duration":0},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528079,"stop":1681596528079,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528079,"stop":1681596528079,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528079,"stop":1681596528079,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528079,"stop":1681596528079,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528079,"stop":1681596528079,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528079,"stop":1681596528079,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528080,"stop":1681596528080,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528080,"stop":1681596528080,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528080,"stop":1681596528080,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528080,"stop":1681596528080,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528080,"stop":1681596528080,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528080,"stop":1681596528080,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681596528080,"stop":1681596528080,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":16,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[{"uid":"b549cee6bb6eea4f","name":"e2e/tests/runAll.cy.ts:2023-04-15T22:08:47.725Z:attempt-1","source":"b549cee6bb6eea4f.png","type":"image/png","size":154851},{"uid":"9797cb3b0a8e8a6d","name":"video recording","source":"9797cb3b0a8e8a6d.mp4","type":"video/mp4","size":3517447}],"parameters":[],"stepsCount":21,"attachmentsCount":2,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"package","value":"e2e.tests.runAll.cy.ts"},{"name":"parentSuite","value":"Vite React Microfrontends"},{"name":"suite","value":"It checks host app"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"7e049908be54748c","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681596525846,"stop":1681596526823,"duration":977}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":32,"broken":0,"skipped":0,"passed":111,"unknown":0,"total":143},"items":[{"uid":"dd063f2e256bb43b","reportUrl":"https://module-federation.github.io/module-federation-examples/707//#testresult/dd063f2e256bb43b","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681537398863,"stop":1681537400366,"duration":1503}},{"uid":"837bca5a9240fec2","reportUrl":"https://module-federation.github.io/module-federation-examples/704//#testresult/837bca5a9240fec2","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681519370815,"stop":1681519371816,"duration":1001}},{"uid":"a956bdea614d8ad8","reportUrl":"https://module-federation.github.io/module-federation-examples/696//#testresult/a956bdea614d8ad8","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681338396087,"stop":1681338397721,"duration":1634}},{"uid":"3867b546c1586ae1","reportUrl":"https://module-federation.github.io/module-federation-examples/695//#testresult/3867b546c1586ae1","status":"passed","time":{"start":1681324155845,"stop":1681324156873,"duration":1028}},{"uid":"8aee644f758dbaa5","reportUrl":"https://module-federation.github.io/module-federation-examples/694//#testresult/8aee644f758dbaa5","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681323452137,"stop":1681323453478,"duration":1341}},{"uid":"edfe88960ab3e81a","reportUrl":"https://module-federation.github.io/module-federation-examples/693//#testresult/edfe88960ab3e81a","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681299802379,"stop":1681299804003,"duration":1624}},{"uid":"84d3fa49363aec2","reportUrl":"https://module-federation.github.io/module-federation-examples/687//#testresult/84d3fa49363aec2","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681214568271,"stop":1681214569713,"duration":1442}},{"uid":"997de2becc573fd0","reportUrl":"https://module-federation.github.io/module-federation-examples/686//#testresult/997de2becc573fd0","status":"passed","time":{"start":1681146964157,"stop":1681146965312,"duration":1155}},{"uid":"27373e5847d1e9ea","reportUrl":"https://module-federation.github.io/module-federation-examples/685//#testresult/27373e5847d1e9ea","status":"passed","time":{"start":1681146964157,"stop":1681146965312,"duration":1155}},{"uid":"fe1d5d55536f441d","reportUrl":"https://module-federation.github.io/module-federation-examples/684//#testresult/fe1d5d55536f441d","status":"passed","time":{"start":1681136328138,"stop":1681136328995,"duration":857}},{"uid":"96abb84145c96e65","reportUrl":"https://module-federation.github.io/module-federation-examples/683//#testresult/96abb84145c96e65","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681099500835,"stop":1681099502063,"duration":1228}},{"uid":"59a14d4544fdf485","reportUrl":"https://module-federation.github.io/module-federation-examples/682//#testresult/59a14d4544fdf485","status":"passed","time":{"start":1681098136259,"stop":1681098137117,"duration":858}},{"uid":"19447979c36349ee","reportUrl":"https://module-federation.github.io/module-federation-examples/671//#testresult/19447979c36349ee","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680987647143,"stop":1680987648420,"duration":1277}},{"uid":"48db0c64cbe14362","reportUrl":"https://module-federation.github.io/module-federation-examples/665//#testresult/48db0c64cbe14362","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680909799420,"stop":1680909800449,"duration":1029}},{"uid":"d99fe0859479fa6e","reportUrl":"https://module-federation.github.io/module-federation-examples/661//#testresult/d99fe0859479fa6e","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680834516060,"stop":1680834517994,"duration":1934}},{"uid":"37ddaae828059c1c","reportUrl":"https://module-federation.github.io/module-federation-examples/657//#testresult/37ddaae828059c1c","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680806534568,"stop":1680806535877,"duration":1309}},{"uid":"be2874ebfde12614","reportUrl":"https://module-federation.github.io/module-federation-examples/630//#testresult/be2874ebfde12614","status":"passed","time":{"start":1680739141211,"stop":1680739142295,"duration":1084}},{"uid":"2de2f2a34c5a1a20","reportUrl":"https://module-federation.github.io/module-federation-examples/619//#testresult/2de2f2a34c5a1a20","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680701535430,"stop":1680701537506,"duration":2076}},{"uid":"fa84eddf26cbdde7","reportUrl":"https://module-federation.github.io/module-federation-examples/616//#testresult/fa84eddf26cbdde7","status":"passed","time":{"start":1680695668573,"stop":1680695669541,"duration":968}},{"uid":"9716e76161237cc7","reportUrl":"https://module-federation.github.io/module-federation-examples/615//#testresult/9716e76161237cc7","status":"passed","time":{"start":1680695668573,"stop":1680695669541,"duration":968}}]},"tags":[]},"source":"7cf890e9dd9ecea1.json","parameterValues":[]}